base/nvti.c

```diff
@@ -1057,6 +1057,46 @@ nvti_set_solution_type (nvti_t *n, const gchar *solution_type)
   return (0);
 }
 
+/**
+ * @brief Add a tag to the NVT tags.
+ *
+ * @param n The NVT Info structure.
+ *
+ * @param name  The tag name. A copy will be created from this.
+ * @param value The tag value. A copy will be created from this.
```

Comment:
```suggestion

 * @param value The tag value. A copy will be created from this.
```

Reply:
Hm, I rather preferred to remove the extra empty line to make it consistent.

Reply:
Consistent with what?  :)

The rest of the file uses spacing (except for two functions right at the end).

The rest of gvm-libs does not use the spacing though.

Reply:
Will keep in mind for new PR.